pkg/graph/schema/types/analyzed_audit.graphql

```diff
@@ -51,7 +51,7 @@ type AnalyzedPlanSections {
     updated: [TableName!]!
     readyForReview: [TableName!]!
     readyForClearance: [TableName!]!
-
```

Comment:
Would this mean we're saying:
- `null`: "Data exchange wasn't updated _at all_"
- `false`: "Data exchange was updated, but wasn't marked as complete"
- `true`: "Data exchange was updated, and was marked complete"

If that's the case, I almost think just leaving it as a non-nullable boolean might be easiest, since we only ever have 2 cases to deal with, as far as I know:
- Don't show the "Data exchange is complete" bullet (`false`)
- _Do_ show the "Data exchange is complete" bullet (`true`)

Reply:
My biggest issue with a non-nullable bool is that it isn't completely accurate to the data being returned returned. The analyzed audit in the DB doesn't have an entry for dataExchangeMarkedComplete except when it is true. 

Perhaps that's a non-nullable ok functionally, since we are just showing and hiding a bullet. 
But, I know I was a bit confused seeing `false`, 
- (does it mean that it wasn't marked complete?) 
- (Did it get changed from being marked complete to in progress?)

 I'd rather say it was marked complete, or not say anything about it.

Reply:
@patrickseguraoddball Could you weigh in on frontend preference for this field?

Reply:
We did a stay-behind in stand up but we're all set on this (leaving it Nullable!)